### ISSUE TITLE
node startup request l1 interface timeout

### ISSUE THREAD

**Issue by wangjiangw** [0xPolygonHermez/zkevm-node#2370]
hi all，
recently, i had construct a production node in my server, but when i startup i found it always failed. after analysis, i found in zkevm-sync startup, it will get logs from l1 point, that is :
![image](https://github.com/0xPolygonHermez/zkevm-node/assets/7013043/c5c5f2cc-a7ea-467e-9419-d672bdd5315c)

but the GenesisBlockNum is hard-coded, current is 
![image](https://github.com/0xPolygonHermez/zkevm-node/assets/7013043/2f663063-f967-4501-8844-d82c4c72d1eb)

but it is too low than current block height in l1 network, so the request will be refused or execute timeout, and then cause startup failed.

so i hava some questions:
1、any new version testnetgenesis we can use ?
2、how to process this situation now ? any configuration can be used ?

**Comment by fanbsb** [comment]
Same issue met, I suggest the team could hardcode the fork id in the genesis file, to avoid loading the data from L1 during startup.

**Comment by wangjiangw** [comment]
> Same issue met, I suggest the team could hardcode the fork id in the genesis file, to avoid loading the data from L1 during startup.

yes, i have to adjust timeout config for l1 node, maybe cost 3 minutes to start up successfully

**Comment by fanbsb** [comment]
3 minutes is a quite long time for the timeout, is that query required at init boot only? what about after the startup or during the reboot?

**Comment by wangjiangw** [comment]
> 3 minutes is a quite long time for the timeout, is that query required at init boot only? what about after the startup or during the reboot?

@fanbsb it looks just in the init phase, reboot too.
